components/menu/menu-item-mixin.js

```diff
@@ -91,6 +91,9 @@ export const MenuItemMixin = superclass => class extends superclass {
 	}
 
 	__initializeItem() {
+		if (this.disabled) {
+			this.setAttribute('aria-disabled', 'true');
```

Comment:
Should we do this in `updated()` instead so that changes to the property are reflected?

Reply:
Could also have a private `_ariaDisabled` that binds to `aria-disabled` that we set from `updated()` -- that would avoid modifying DOM outside of render.

Reply:
Agreed on the changing in `updated`. I'm not totally clear on what the suggestion is about binding to `aria-disabled` since `menu-item-mixin` doesn't have a `render` and it seems a bit complex to modify in each component that uses this mixin.

Reply:
I just meant it could have a property:
```
_ariaDisabled: { type: String, attribute: 'aria-disabled' }
```

And then in the `updated()` for `disabled` it could go `this._ariaDisabled = this.disabled ? 'true' : 'false'` which would result in the same as going `this.setAttribute('aria-disabled', 'true')` except that you'd be letting Lit handle it for you.